code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -114,12 +114,13 @@
 		stored_units = store_mobs(stored_units)
 		RegisterSignal(src, COMSIG_CLOSET_FLASHBANGED, PROC_REF(flashbang))
 
-	opened = 0
+	opened = FALSE
 	update_icon()
 
 	playsound(src.loc, close_sound, 15, 1)
-	density = TRUE
-	return 1
+	if(pixel_y < 10 || pixel_x < 10) // If it's offsetted to a point where it would confuse players, don't make dense
+		density = TRUE
```

Comment:
intended for edge cases, any pixel offset past a certain point communicates to players that it is passable. This wouldn't work if you put it 10 pixels back against a wall and don't adjust X or want to adjust it to be next to something under 10 pixels away for aesthetic

Reply:
open to adjusting pixel amounts though as I am unsure of the point where most people consider it "passable"

Reply:
Sorry for the delay! What I meant was that it currently only makes the closet passable if *both* variables are greater than ten, since that's the 'fail' state of the check:

https://github.com/cmss13-devs/cmss13/assets/57483089/62995738-2bbc-425b-8537-b88679559ef7

(The closet is mapedited to have `pixel_y = 25`, `density = 0`)
<hr>

For comparison of how both versions handle different `pixel_x` and `pixel_y` values:
<table>
  <tr>
    <th></th>
    <th><code>pixel_y &lt; 10 || pixel_x &lt; 10</code></th>
    <th><code>pixel_y &lt; 10 &amp;&amp; pixel_X &lt; 10</code></th>
  </tr>
  <tr>
    <th>Both variables &lt; 10</th>
    <td><code>density = TRUE</code></td>
    <td><code>density = TRUE</code></td>
  </tr>
  <tr>
    <th>One variable &gt;= 10</th>
    <td><code>density = TRUE</code></td>
    <td><code>density = FALSE</code></td>
  </tr>
  <tr>
    <th>Both variables &gt;= 10</th>
    <td><code>density = FALSE</code></td>
    <td><code>density = FALSE</code></td>
  </tr>
  </table>